cmd/maestro/environments/framework.go

```diff
@@ -164,6 +168,13 @@ func (e *Env) LoadClients() error {
 		return err
 	}
 
+	cloudEventsSourceOptions := mqttoptions.NewSourceOptions(e.Config.MessageBroker.MQTTOptions, e.Config.MessageBroker.SourceID)
+	e.Clients.CloudEventsSource, err = cloudevents.NewSourceClient(cloudEventsSourceOptions, e.Services.Resources())
```

Comment:
there is the `LoadClients` func.  probably better to include this client init in that function instead of this higher level init func.

Reply:
If you expand the code, you'll discover that the CloudEvents source client is initialized within the `LoadClients` function, similar to the OCM client.